tests/benchdnn/softmax/softmax.hpp

```diff
@@ -74,15 +76,16 @@ struct prb_t : public prb_dims_t {
         : prb_t(s.prb_dims, s.dir[0], s.sdt[0], s.ddt[0], s.stag[0], s.dtag[0],
                 s.alg[0], s.axis[0], s.mb[0], s.inplace[0],
                 s.attributes.front(), s.ctx_init[0], s.ctx_exe[0],
-                s.impl_filter) {
+                s.impl_filter, s.has_stats[0]) {
         SAFE_V(s.has_single_setup() ? OK : FAIL);
     }
 
     prb_t(const prb_dims_t &prb_dims, dir_t dir, dnnl_data_type_t sdt,
             dnnl_data_type_t ddt, const std::string &stag,
             const std::string &dtag, alg_t alg, int axis, int64_t mb,
             bool inplace, const attr_t &attr, const thr_ctx_t &ctx_init,
-            const thr_ctx_t &ctx_exe, const impl_filter_t &impl_filter)
+            const thr_ctx_t &ctx_exe, const impl_filter_t &impl_filter,
+            bool has_stats)
```

Comment:
Nit: please put between the `axis` and `mb`. Last several entries are unified across all drivers and are used for the fast insert through sed when a massive driver update is needed. Breaking the pattern will make it harder to update all at once.